geoai/universat.py

```
@@ -0,0 +1,209 @@
+        **kwargs,
+    ):
+        batch = self.format_batch(
+            [samples] if isinstance(samples, dict) else samples, normalize_scales
+        )
+        return self.model.encode(
+            batch, patch_size=patch_size, output_grid=output_grid, **kwargs
+        )
+
+
+def universat_inference(
+    samples, patch_size=40.0, output_grid=None, device=None, **kwargs
```

Ruff (0.15.21)

[warning] Missing type annotation for `**kwargs`

(ANN003)